src/libs/polycube/src/common.cpp

```diff
@@ -108,4 +108,38 @@ PortStatus string_to_port_status(const std::string &status) {
   }
 }
 
+std::string get_kernel_release() {
+  struct utsname buffer = {};
+  if (uname(&buffer) != 0)
+    throw std::runtime_error("Unable to execute `uname` command");
+  return buffer.release;
+}
+
+bool compare_kernel_release(std::string &current, const std::string& required) {
+  //  Variables to store numeric parts of the release
+  int vnum1 = 0, vnum2 = 0;
+
+  //  Loop until both string are processed. At each loop increase indexes, reset
+  //  numeric parts and go on parsing
+  for (int i=0,j=0; (i<current.length() && j<required.length()); i++, j++, vnum1=vnum2=0){
+    //  storing numeric part of current version in vnum1
+    while (i < current.length() && isdigit(current[i])) {
+      vnum1 = vnum1 * 10 + (current[i] - '0');
+      i++;
+    }
+
+    //  storing numeric part of required version in vnum2
+    while (j < required.length() && isdigit(required[j])) {
+      vnum2 = vnum2 * 10 + (required[j] - '0');
+      j++;
+    }
+
+    if (vnum1 > vnum2)
+      return true;
+    if (vnum2 > vnum1)
+      return false;
+  }
+  return true;
+}
+
```

Comment:
@s41m0n If I am not wrong, in Polycube utils there was already a function to check the current kernel version [1].
It would be better to use that one and avoid duplicate code.

https://github.com/polycube-network/polycube/blob/83b4a6d6720b6bf91caf63f3a89f9800f3b847e9/src/polycubed/src/utils/utils.cpp#L28

Reply:
https://github.com/polycube-network/polycube/blob/83b4a6d6720b6bf91caf63f3a89f9800f3b847e9/src/polycubed/src/utils/utils.cpp#L28

Found, I'm going to use that one, thanks!

Reply:
Solved in 523ff8b